ckanext/canada/tests/test_trim_package.py

```diff
@@ -1,48 +1,44 @@
-{"ready_to_publish": false, "catalog_type": "Data | Donn\u00e9es", "title_fra": "Indicateurs de la qualit\u00e9 de l'eau - La qualit\u00e9 de l'eau douce aux stations de suivi  de 2007 \u00e0 2009, au Canada", "keywords": "aaa", "id": "00826755-9718-4285-a1d5-97cfba42626c", "subject": "Nature and Environment  Nature et environnement", "spatial_representation_type": "", "maintenance_and_update_frequency": "Unknown | Inconnu", "title": "Water Quality Indicators - Freshwater quality at monitoring stations for the 2007 to 2009 period, Canada", "author_email": "open-ouvert@tbs-sct.gc.ca", "geographic_region": "", "notes_fra": "b", "presentation_form": "", "spatial": "", "license_id": "ca-ogl-lgo", "resources": [], "browse_graphic_url": "", "topic_category": "", "date_published": "2012-04-05", "time_period_coverage_start": "1000-01-01", "time_period_coverage_end": "3000-01-01", "language": "Bilingual (English and French) | Bilingue (Anglais et Fran\u00e7ais)", "date_modified": "", "notes": "b", "owner_org": "49E2ADF4-AD7A-43EB-85C8-6433D37ED62C", "keywords_fra": "aaa", "resource_type": "file"}
-"""
+    def setup_method(self, method):
+        """Method is called at class level before EACH test methods of the class are called.
+        Setup any state specific to the execution of the given class methods.
+        """
+        reset_db()
+        clear_all()
+
+        self.example_pkg = Dataset()
+        resources = []
+        for _ in range(6):
```

Comment:
don't use `_` as a variable name, it shadows the gettext function `_` in common use

Reply:
variable has been renamed.